### DISCUSSION TITLE
Empty folder should be removed.

### DISCUSSION THREAD

**Discussion by XuanhuiHu** [grafana/grafana#46274] [Dashboards, Panels & Visualizations]
When all the dashboards in a folder had been removed, there is a empty folder show permanently.
Can we remove this empty folder?

**Comment by XuanhuiHu** [comment]
select * from dashboard b where is_folder=1 and not exists (select 0 from dashboard a where a.folder_id=b.id);

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
